src/components/about/content-provenance-he.jsx

```diff
@@ -34,24 +34,27 @@ function Content() {
         <div className="right-column">
           <img alt="Cambridge" src={ImgCambridge} />
         </div>
-        <h3>ספריית מקור: קיימברידג'</h3>
+        <h3>'קיימברידג</h3>
         <p className="text">
           בשנים 1896-1897 נסע ד"ר סלומון שכטר מקיימברידג' לפסטאט כדי לבחון את הגניזה, במימונו של  צ'ארלס טיילור מסט. ג'ון קולג'. שכטר קיבל את רשותה של הקהילה היהודית במצרים לקחת משם כל מה שימצא חן בעיניו ('הכל מצא חן בעיני', אמר מאוחר יותר), והוא הביא עמו לקיימברידג' 193,000 קטעי גניזה, המרכיבים את אוסף טיילור-שכטר (T-S Collection).
         </p>
         <p className="text">
 לצד אוסף טיילור-שכטר, שוכן כיום בקיימברידג' גם אוסף מוצרי, המונה 7000 קטעי גניזה. קטעים אלה, שנאספו על ידי איש העסקים הקהירי המצליח ג'ק מוצרי בעשור הראשון של המאה העשרים, נועדו להישמר במצרים כחלק ממורשתה התרבותית של הקהילה היהודית המקומית. אך בעקבות מותו ללא עת של מוצרי בשנת 1934 עזבה המשפחה את מצרים, והאוסף הקרוי על שמו נעלם מעיני הציבור עד שנות השבעים, אז צולם למיקרופילם על ידי צוות של הספרייה הלאומית בירושלים. בעקבות סיכומים חדשים עם בני משפחת מוצרי, הושאל האוסף  בשנת 2006 לקיימברידג' למשך 20 שנה. במהלך תקופה זו יעבור  האוסף שימור, דיגיטציה וקיטלוג מפורט.
         </p>
         <p className="text">
-כיום, זמינים ברשת יותר מ-18,000 קטעי גניזה מאוספי טיילור-שכטר, CUL Or. ומוצרי, ביניהם מסמכים רבים (מכתבים ומסמכים משפטיים) וקטעי ליטורגיה (פרי שיתוף הפעולה עם מפעל התפילה באוניברסיטת בן-גוריון). קטעים נוספים יועלו לרשת באופן סדיר, עד שיהיה כולו זמין לציבור.
+כיום, זמינים ברשת יותר מ-18,000 קטעי גניזה מאוספי טיילור-שכטר, CUL Or.
+ומוצרי, ביניהם מסמכים רבים (מכתבים ומסמכים משפטיים) וקטעי ליטורגיה (פרי שיתוף הפעולה עם מפעל התפילה באוניברסיטת בן-גוריון). קטעים נוספים יועלו לרשת באופן סדיר, עד שיהיה כולו זמין לציבור. למידע נוסף בקרו באתר
+          <p className="text">
```

Comment:
This is a paragraph in a paragraph? Let me check how it looks like on render and/or compare with English/Arabic

Reply:
OK, the `<p><p>Paragraph Inception</p></p>` wasn't intended. Pushing a fix shortly. Pretty minor though, so no worries.